demonstrations/tutorial_classical_expval_estimation.py

```diff
@@ -1,6 +1,6 @@
 r"""
-Classically estimating expectation values from parametrized quantum circuits
-============================================================================
+Pauli Propagation: Classically estimating expectation values from parametrized quantum circuits
+============================================================================================
```

Comment:
Two things about this:
1. sorry, my suggestion still seems to have had the wrong length (the line of equality signs is supposed to have the same length as the title length, as far as I know).
2. it doesn't seem to matter and renders fine, as far as I can tell :shrug: :laughing:

Reply:
```suggestion
==============================================================================================
```

that makes it match :) Note that while it will render fine, it will generate warnings in our build system